contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -98,26 +84,18 @@ export default {
     updateFullName(context, { first_name, last_name }) {
       let currentUser = context.state.currentUser;
       currentUser = { ...currentUser, first_name, last_name };
-      context.commit('SET_CURRENT_USER', currentUser);
+      context.commit('UPDATE_CURRENT_USER', currentUser);
     },
-    fetchDeferredUserData(context, settings = false) {
-      if (context.getters.availableSpace) {
-        if (
-          (context.getters.storageUseByKind && context.state.currentUser.api_token) ||
-          !settings
-        ) {
-          return;
-        }
-      }
-      let promise;
-      if (settings) {
-        promise = settingsDeferredUser();
-      } else {
-        promise = deferredUser();
-      }
-      return promise.then(response => {
-        context.commit('UPDATE_CURRENT_USER', response.data);
+    fetchUserStorage: debounce(function(context) {
+      return client.get(window.Urls.user_get_storage_used()).then(response => {
+        context.commit('UPDATE_CURRENT_USER', { disk_space_used: response.data });
```

Comment:
We might want to update the current user in the indexedDB table here too? So that other tabs also get this update.